gitops/base-install/ebs-csi-driver/install.yaml

```diff
@@ -17,12 +17,31 @@ metadata:
     - resources-finalizer.argocd.argoproj.io
 spec:
   project: admins
-  source:
-    chart: aws-ebs-csi-driver
+  sources:
+  - repoURL: 'https://github.com/pelotech/foundation'
+    path: 'gitops/base-install/ebs-csi-driver/snapshot-controller'
+    targetRevision: main
+  - chart: aws-ebs-csi-driver
     repoURL: https://kubernetes-sigs.github.io/aws-ebs-csi-driver
-    targetRevision: 2.25.0
+    targetRevision: 2.26.0
     helm:
       releaseName: aws-ebs-csi-driver
+      values: |
+        storageClasses:
+        - name: gp3 
+          annotations:
+            storageclass.kubernetes.io/is-default-class: "true"
+          volumeBindingMode: WaitForFirstConsumer
+          reclaimPolicy: Delete
+          parameters:
+            type: gp3
+          # TODO: Encrypting volumes by default seems desirable, but may have unintended consequences
+          #     encrypted: "true"
+        volumeSnapshotClasses:
+        - name: gp3
+          annotations:
+            snapshot.storage.kubernetes.io/is-default-class: "true"
```

Comment:
Just to keep in mind if gp3 is the new default, VMs should reference their own storage class to avoid issue mentioned [here](https://pelotech.slack.com/archives/C05TEM6CE0Z/p1702695725502829) or investigating more the root cause of this "tsc frequency" custom request appearing Karpenter can't handle